app/views/devise/registrations/new.html.slim

```diff
@@ -70,3 +113,24 @@
               .d-flex.justify-content-center.links.mt-2
                 = t('devise.registrations.new.have_account_text')
                 = render 'devise/shared/links'
+script src="https://code.jquery.com/jquery-1.12.4.js" 
+javascript:
+  $(document).on("click", "[type='checkbox']", function(e) {
+        if (this.checked) {
+            $(this).attr("value", "true");
+        } else {
+            $(this).attr("value","false");}
+    });
+  function myFunction() {
+    if($('#flag').prop('checked')) {
+          $('#OsbbName').addClass('d-flex');
```

Comment:
can you just $('#OsbbPhone, #OsbbEmail').addClass('d-flex')
DRY
also string with IDs you can move to varaible, and reuse in if/else blocks

Reply:
Ok,refactor